matchbox_socket/src/webrtc_socket/wasm.rs

```diff
@@ -79,10 +76,10 @@ impl Signaller for WasmSignaller {
 }
 
 impl PeerDataSender for RtcDataChannel {
-    fn send(&mut self, packet: Packet) -> Result<(), MessagingError> {
+    fn send(&mut self, packet: Packet) -> Result<(), MessageSendError> {
         self.send_with_u8_array(&packet)
             .efix()
-            .map_err(MessagingError::from)
+            .map_err(MessageSendError::from)
```

Comment:
Would use the `Ok(X?)` pattern here too

Reply:
Not sure what you mean?

Reply:
I think this is outdated now so I'll resolve